src/Html/Button.php

```diff
@@ -183,7 +185,7 @@ public function actionSubmit()
      */
     public function action($value)
     {
-        $this->attributes['action'] = $value;
+        $this->attributes['action'] = "function(e, dt, node, config) { $value }";
```

Comment:
I think we should check first if the value starts with `function()`. I have some implementation that set the custom action function and this would break those codes.

Reply:
I improve the code. https://github.com/yajra/laravel-datatables-html/pull/114/commits/2abe751458f22a03b54160bc2cb466a09fb12c4b